core/templates/components/summary-tile/learner-topic-summary-tile.component.html

```diff
@@ -1,27 +1,45 @@
-<mat-card class="oppia-topic-dashboard-card">
-  <a [href]="getTopicLink()"
-     target="{{ openInNewWindow ? '_blank' : '_self' }}"
-     class="oppia-topic-link">
-    <div class="oppia-title-section" [ngStyle]="{'background-color': thumbnailBgColor}">
-      <!-- We add an empty alt text so that the screen reader does not read the thumbnail image filename and ignores it as the thumbnail does not convey any significant information. -->
+<ng-container>
+  <mat-card *ngIf="!featureFlag" class="oppia-topic-dashboard-card">
+    <a [href]="getTopicLink()"
+       target="{{ openInNewWindow ? '_blank' : '_self' }}"
+       class="oppia-topic-link">
+      <div class="oppia-title-section" [ngStyle]="{'background-color': thumbnailBgColor}">
+        <!-- We add an empty alt text so that the screen reader does not read the thumbnail image filename and ignores it as the thumbnail does not convey any significant information. -->
+        <img alt="" class="oppia-thumbnail-image" [src]="thumbnailUrl">
+      </div>
+      <div class="oppia-summary-section">
+        <span class="oppia-topic-title e2e-test-learner-topic-summary-tile-title">
+          {{topicTitle}}
+        </span>
+        <div class="oppia-topic-lessons">
+          {{totalPublishedNodeCount}}
+          <div *ngIf="totalPublishedNodeCount === 1" class="lesson-title">
+            <span translate="I18N_TOPIC_VIEWER_LESSON"></span>
+          </div>
+          <div *ngIf="totalPublishedNodeCount !== 1" class="lesson-title">
+            <span translate="I18N_TOPIC_VIEWER_LESSONS"></span>
+          </div>
+        </div>
+      </div>
+    </a>
+  </mat-card>
+  <!--REDESIGN-->
```

Comment:
Here and elsewhere, can we remove the comments that just say "redesign"?

Reply:
Done